src/daemon/start_rgw.sh

```diff
@@ -29,8 +29,14 @@ function start_rgw {
   log "SUCCESS"
 
   local rgw_frontends="civetweb port=$RGW_CIVETWEB_IP:$RGW_CIVETWEB_PORT"
-  if [ "$RGW_REMOTE_CGI" -eq 1 ]; then
-    rgw_frontends="fastcgi socket_port=$RGW_REMOTE_CGI_PORT socket_host=$RGW_REMOTE_CGI_HOST"
+
+  if [ ! -f "$RGW_SSL_CERTIFICATE" ]; then
+    # Suffix 's' on port required for SSL
+    rgw_frontends="${rgw_frontends}s ssl_certificate=${RGW_SSL_CERTIFICATE}"
+  else
```

Comment:
I don't know all of the RGW config options, but I guess this makes these two options mutually exclusive, is intentional?

Reply:
You're right, although `RGW_REMOTE_CGI` is not encouraged and I wonder if we should not remove it.

Reply:
It was intentional. They are mutually exclusive because rgw_frontends will be clobbered if RGW_REMOTE_CGI is true which overwrites the constructed SSL configuration.

The `else` can be removed, but that will just result in rgw_frontends being overwritten with CGI configuration if both are provided.